docs/book/src/clusterctl/commands/move.md

```diff
@@ -58,6 +58,15 @@ make it clear those operation have the same limitations of the move command.
 
 </aside>
 
+<aside class="note warning">
+
+<h1> Warning: Status get lost </h1>
+
+Only the `Spec` of the CRDs will be moved. The `Status` (and `Status.Conditions`) will get lost. 
+Implementors of custom controllers should not store non-ephemeral data in the `Status`.
```

Comment:
The `Status` (and `Status.Conditions`) of resources are lost during `clusterctl move`. Provider implementers should not store non-ephemeral data in the `Status`. `Status` should be able to be fully rebuilt by controllers by observing the current state of resources.

Reply:
@killianmuldoon I updated the wording. Please tell me if something is missing.